src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -9,6 +9,10 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{A55A2B9A-830F-4330-A0E7-02A9FB30ABD2}</ProjectGuid>
     <OutputType>Library</OutputType>
+    <UnsupportedPlatforms>OSX</UnsupportedPlatforms>
```

Comment:
This is going to disable all of our System.Net.Security tests on OS X.  We don't want to do that.  If the new NegotiateStream tests won't yet run on OS X, then they should be explicitly disabled with an ActiveIssue or something similar.